### DISCUSSION TITLE
【文档无法打开】无法打开文档中 端侧部署的环境配置文档

### DISCUSSION THREAD

**Discussion by 136323000** [PaddlePaddle/PaddleOCR#12954] [Q&A] [answered]
#### 问题描述 / Problem Description
https://github.com/PaddlePaddle/PaddleOCR/blob/main/deploy/lite/readme.md
我想按照这个文档自己编译
配置ANDROID_NDK和NDK_ROOT环境变量后，输入指令没有反应 如图：
<img width="873" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/49808133/214beb69-588e-46f1-a0b0-fc360b1ed6fe">

我怀疑是环境问题 
但是这里文档无法打开
<img width="1439" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/49808133/9e463226-23b7-47a1-a302-34c8bcfd415e">
<img width="1341" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/49808133/fd351c59-1aa9-45b3-a0bc-1a2f264b2fe0">

另外 我们有没有官方维护的jiao'liu交流群

**Comment by SWHL** [comment, marked as answer]
文档问题，接下来一段时间会逐步修复

目前只有Discord社区，可以加入进去 https://discord.gg/AX9P8r6q
